configure-thread-border-router/run.sh

```
@@ -0,0 +1,142 @@
+# Display Welcome
+print_header "Big Bear Thread Border Router Communication Setup V0.0.1"
+echo "Here are some links:"
+echo "https://community.bigbeartechworld.com"
+echo "https://github.com/BigBearTechWorld"
+echo ""
+echo "If you would like to support me, please consider buying me a tea:"
+echo "https://ko-fi.com/bigbeartechworld"
+echo ""
+
+# Check kernel configuration for IPV6_ROUTER_PREF
+if grep -q "CONFIG_IPV6_ROUTER_PREF=y" /boot/config-$(uname -r) 2>/dev/null; then
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)